ta4j-examples/src/main/java/ta4jexamples/indicators/IndicatorsToCsv.java

```diff
@@ -108,7 +109,7 @@ public static void main(String[] args) {
          */
         BufferedWriter writer = null;
         try {
-            writer = new BufferedWriter(new FileWriter("indicators.csv"));
+            writer = new BufferedWriter(new FileWriter(new File("target", "indicators.csv")));
```

Comment:
why?

Reply:
Because the `indicators.csv` is a generated file and by Maven's convention everything that is generated must be placed in the `target` directory, without this fix git complains that the file is not being tracked.

Reply:
ah, I wondered myself sometimes about this behaviour...